docs/src/content/docs/challenges/angular/46-simple-animations.md

```diff
@@ -0,0 +1,40 @@
+---
+title: 🟢 Simple Animations
+description: Challenge 46 is about learning Angular's integrated animation API
+author: sven-brodny
+challengeNumber: 46
+command: angular-simple-animations
+sidebar:
+  order: 17
+  badge: New
+---
+
+## Information
+
+This is the first of two animation challenges, the goal of this series is to master animations in Angular.
+
+Well-designed animations can make your application more fun and straightforward to use, but they aren't just cosmetic. Animations can improve your application and user experience in a number of ways:
+
+- Without animations, web page transitions can seem abrupt and jarring.
+- Motion greatly enhances the user experience, so animations give users a chance to detect the application's response to their actions.
+- Good animations intuitively call the user's attention to where it is needed.
+
+I would recommend you read the [official documentation](https://angular.io/guide/animations). You will learn everything that is necessary to successfully complete the challenge.
+
+Otherwise look at this [working example](https://svenson95.github.io/ng-xmp-animations/) and [git repo](https://github.com/svenson95/ng-xmp-animations) to get inspired.
+
+## Statement
+
+The goal of this challenge is to add animations.
+
+## Constraints
+
+Don't use any CSS and utilize Angular's integrated `@angular/animations` API.
+
+### Level 1
+
+Add a fading or moving animation for the paragraphs on the left side.
+
+### Level 2
+
+Add a stagger animation for the list on the right side.
```

Comment:
I would to have a video on how to final result should look like. It's easier to understand what we should do

Reply:
Yes you're right, i'll update the docs to make this more understandable. 

Thanks for creating this project! The challenges are really nice and helpful, hope i can contribute more in future 😊

Reply:
changes done!

Reply:
Ok that's way better. 👍

But for the video, if you have already done the challenge, you could show the final result by recording your screen. 
But that's fine for me. I'll merge it tomorrow.
And then, you can submit your solution in one or two weeks.

Great job. Thanks a lot

Reply:
Perfect. I think it's better if we don't show the final result in advance. As soon as this PR is merged i'll build my solution and add a final result video in the solution pull request.

Reply:
I disagree on this point. Having the final result is always better. 
Wen you work in a company, the design team or product team shows the desired result in a mock template like figma and you can implement it afterward. 
But I will not block the PR for it.

Reply:
Okay then i'll add the final result to the docs when i've got the solution done. Thanks for the feedback :)